pkg/controller/statefulset/stateful_pod_control.go

```diff
@@ -194,6 +194,8 @@ func (spc *realStatefulPodControl) createPersistentVolumeClaims(set *apps.Statef
 		case err != nil:
 			errs = append(errs, fmt.Errorf("failed to retrieve PVC %s: %s", claim.Name, err))
 			spc.recordClaimEvent("create", set, pod, &claim, err)
+		case pvc.DeletionTimestamp != nil:
```

Comment:
the switch is on the error states so I'd prefer explicit
```
case err=nil:
    if pvc.DeletionTimestamp != nil
```

Reply:
Fixed as suggested.